src/Extension/ExtensionManager.php

```diff
@@ -138,10 +138,18 @@ public function disable($name)
         $enabled = $this->getEnabled();
 
         if (($k = array_search($name, $enabled)) !== false) {
-            unset($enabled[$k]);
-
             $extension = $this->getExtension($name);
 
+            if (in_array('locale', $extension->keywords)) {
```

Comment:
But that won't be a good extension (language pack), will it?
Maybe a better way is to check 'locale' in keywods first, and then look for 'flarum-locale' if 'locale' was not found. Since this action will happed not that often, it won't take up resources.